cmd_actuators_mux/param/actuator_mux.yaml

```diff
@@ -0,0 +1,36 @@
+# Individual subscriber configuration:
+#   name:           Source name
+#   topic:          The topic that provides  messages
+#   timeout:        Time in seconds without incoming messages to consider this topic inactive
+#   priority:       Priority: an UNIQUE unsigned integer from 0 (lowest) to MAX_INT 
+#   short_desc:     Short description (optional)
+
+
+# Fin Angles
+fin_angles_subscribers:
+  - name:        "Autopilot"
+    topic:       "input/autopilot/set_angles"
+    timeout:     0.1
+    priority:    0
+    short_desc:  "Command sent by the Autopilot controller to the fin controller"
+  - name:        "Jaus Ros Bridge"
+    topic:       "input/jausRosBridge/set_angles"
+    timeout:     0.5
+    priority:    1
+    short_desc:  "Commands sent by the Jaus ROS Bridge to the fin controller"
+fin_angles_publisher:       "fin_control/set_angles"
+
+
+# Thruster
+set_rpm_subscribers:
+  - name:        "Autopilot"
+    topic:       "input/autopilot/set_rpm"
+    timeout:     0.1
+    priority:    0
+    short_desc:  "Command sent by the Autopilot controller to the Thruster Controller"
+  - name:        "Jaus Ros Bridge"
+    topic:       "input/jaus_ros_bridge/set_rpm"
+    timeout:     0.5
+    priority:    1
+    short_desc:  "Command sent by the Jaus ROS Bridge to the Thruster Controller"
+set_rpm_publisher:       "thruster_control/set_rpm"
```

Comment:
@patapufate meta: what if the names in each group do not match?

Reply:
Added assertion in bf8da9c